packages/ignite-cli/src/commands/plugin.js

```diff
@@ -105,29 +106,110 @@ Generally, you would run this from ./YourApp/ignite/plugins/
 Commands:
   ignite plugin help
   ignite plugin new <your-plugin>
-  ignite plugin list (coming soon)
+  ignite plugin list
+  ignite plugin search <search string>
+  ignite plugin info <plugin name>
```

Comment:
Wouldn't it be better to use a centralized server like npm or bundler? That way you don't have to push code every time someone creates a cool plugin. Who's up for the challenge?